common/koa-middleware/withPopupDialog.js

```diff
@@ -0,0 +1,29 @@
+const Prismic = require('prismic-javascript');
+
+let popupDialog = {
+  openButtonText: null,
+  dialogHeading: null,
+  dialogCopy: [{ type: 'paragraph', text: null, spans: null }],
+  ctaText: null,
+  ctaLink: { url: null },
+  isShown: false,
+};
+
+async function getAndSetPopupDialog() {
+  try {
+    const api = await Prismic.getApi(
+      'https://wellcomecollection.prismic.io/api/v2'
```

Comment:
api.js has apiUri . we could reuse this or make this exposed

Reply:
Deffo.

Reply:
…aaaactually, this is a CommonJS module (`require`) not an es6 module (`import`), so not as straightforward as I'd hoped. Which is also possibly why this isn't the only place this has happened in the middleware functions.